tests/end-to-end/backstop.js

```diff
@@ -63,20 +66,23 @@ const scenarios = spaces.flatMap(space => [
   { space: space, page: 'navbar', selectors: [selectors.header] },
   { space: space, page: 'navbar', selectors: [selectors.body],
     hoverSelector: '.navbar-item.has-dropdown.is-hoverable',
-    viewports: viewports.filter((viewport) => ['lg', 'xl', 'mx'].includes(viewport.label))
+    viewports: viewports.filter((viewport) => ['lg', 'xl', 'mx'].includes(viewport.label)),
+    postInteractionWait: postHoverInterval,
```

Comment:
There is no direct analogy in Backstop.js, but I will try to combine 'postInteractionWait' with 'readySelector' option - then we'll wait unti element with given selector appears on the page. But we still need a timeout in case it does not appear.

Reply:
I think it's a different case than what you had in Selenium. We had something like you described in Cypress tests.  Here we could not define on what we are waiting for (page should be rendered, but what does it mean?), so Backstop simply waits for a given interval and does the snapshot. In other words, this `postInteractionWait` means "if there were no interactions for the last 1s then the page is supposed to be ready".

Reply:
And of course, that's why it's unstable :/

Reply:
And, 'readySelector' didn't work out, so I leave it as it is for now.